private/model/api/load.go

```diff
@@ -172,6 +173,9 @@ func (a *API) Setup() {
 	}
 
 	a.fixStutterNames()
+	if err := a.validateShapeNames(); err != nil {
+		log.Fatalf(err.Error())
+	}
```

Comment:
It would be helpful for this `log.Fatalf` to include the name of the API that failing.

Reply:
The error was already wrapped with shape name, added package name to error. 
```go
return fmt.Errorf("attempted to rename shape %v to %v for package %v, but the new name results in shape name collision",s.ShapeName, name, a.PackageName())
```